### ISSUE TITLE
Add Scrolling Bar to Save Menu

### ISSUE THREAD

**Issue by Yumeris** [Vita3K/Vita3K#2304]
its not possible to save more than what seen in the menu
![image](https://user-images.githubusercontent.com/70304827/218287886-0ad72100-0df9-434a-85e9-aad86f641ae9.png)
![image](https://user-images.githubusercontent.com/70304827/218287899-00d357f5-03fd-4e70-9da1-150a2fe0262b.png)

**Comment by Croden1999** [comment]
yes, me too, but we can used mouse wheel or touch pad to scrolling on pc, could not on android.

**Comment by Zangetsu38** [comment]
i see

**Comment by Macdu** [comment]
This will be fixed in the next APK release.

**Comment by Yumeris** [comment]
> This will be fixed in the next APK release.

add it for pc as well?, kinda confusing.. i thought at first its not scrollable..

**Comment by nishinji** [comment]
Outdated
